vignettes/response-FDA IR-pilot3.qmd

```diff
@@ -0,0 +1,243 @@
+    SAFFL    format.sas     NULL          $1      
+     SEX     format.sas     NULL          $1      
+  ------------------------------------------------
+```
+
+{{< pagebreak >}}
+
+***ADAE***
+
+```         
+adae <- read_xpt(file.path(path$adam, "adae.xpt"))
+adae_cdisc <- read_xpt(file.path(path$cdisc, "adae.xpt"))
+
+arsenal::comparedf(adae, adae_cdisc)
+diffdf::diffdf(adae, adae_cdisc, keys = c("STUDYID", "USUBJID", "AESEQ"))
+waldo::compare(adae, x_arg = "old", adae_cdisc, y_arg = "new")
+
+=======================================================================================
+   VARIABLE  ATTR_NAME             VALUES.BASE                      VALUES.COMP          
+---------------------------------------------------------------------------------------
+  ADURN      label          Analysis Duration (N)               AE Duration (N)    
+  
+  ADURU      label         Analysis Duration Units             AE Duration Units      
+  
+  AOCCFL     label    1st Occurrence within Subject ...  1st Occurrence of Any AE Flag 
+---------------------------------------------------------------------------------------
+```
+
+***ADLBC***
+
+```         
+adlbc <- read_xpt(file.path(path$adam, "adlbc.xpt"))
+adlbc_cdisc <- read_xpt(file.path(path$cdisc, "adlbc.xpt"))
+
+arsenal::comparedf(adlbc, adlbc_cdisc)
+diffdf::diffdf(adlbc, adlbc_cdisc, keys = c("STUDYID", "USUBJID", "LBSEQ", 
+"PARAMCD", "AVISIT")) 
+waldo::compare(adlbc, x_arg = "old", adlbc_cdisc, y_arg = "new")
+
+  ==================================
+   VARIABLE  CLASS.BASE  CLASS.COMP 
+  ----------------------------------
+     ADT        Date      numeric   
+    TRTEDT      Date      numeric   
+    TRTSDT      Date      numeric   
+  ----------------------------------
+```
+
+{{< pagebreak >}}
+
+***ADADAS***
+
+```         
+adadas <- read_xpt(file.path(path$adam, "adadas.xpt"))
+adadas_cdisc <- read_xpt(file.path(path$cdisc, "adadas.xpt"))
+
+arsenal::comparedf(adadas, adadas_cdisc)
+diffdf::diffdf(adadas, adadas_cdisc, keys = c("STUDYID", "USUBJID", "QSSEQ", 
+"PARAMCD", "AVISIT"))
+waldo::compare(adadas, x_arg = "old", adadas_cdisc, y_arg = "new")
+
+========================================================================================
+VARIABLE  ATTR_NAME            VALUES.BASE                     VALUES.COMP            
+----------------------------------------------------------------------------------------
+ANL01FL    label            Analysis Flag 01             Analysis Record Flag 01      
+ITTFL      label    Intent-To-Treat Population Flag  Intent-to-Treat Population Flag 
+----------------------------------------------------------------------------------------
+```
+
+Any observed difference in variables is documented in QC findings, including NA versus NULL which we did not encounter. Any observed difference in attribute will not affect the analysis.
+
+In conclusion, the team was unable to identify any differences that could have an impact on the analysis.
```

Comment:
I think we should re-iterate that all these findings are metadata related and have no impact on the analysis

Reply:
@SHAESEN2 and @bms63 ,  let me know what you think of the suggestion.

Reply:
```suggestion
In conclusion, the discrepancies found between the Pilot 3 vs the CDISC ADaMs were all metadata related. The team was unable to identify any of these issues impacting the analysis.
```

Reply:
@laxamanaj will move the respnse paragraph to the top, then update so that it has this wording.